README.md

```diff
@@ -13,6 +13,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 
 HOW TO USE IT
 ---------------
+### Getting Started Guide
+Check out our new [Getting Started](GettingStarted.md) guide if you want a quick introduction to the project!
```

Comment:
But this is the GettingStarted guide? Why advertise yourself?

Reply:
This is in the README.md file

Reply:
Oops, my bad